samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.Ignore();
```

Comment:
I don't have any issue with Skip being a top level supported flag for all events, but it sure isn't critical to have now, we can keep it where we had it before, but we certainly are not going to block over this.

Given that we normalize Skip/Ignore in RemoteAuthenticationResult anyways, I think we should just combine these.  Do any tests even fail if I combine the two?